### PR TITLE
added recipe for transcov

### PR DESCRIPTION
**Pull request by Hogfeldt**
Transcov is a bioinformatics software for mapping coverage around Transcription Start Sites (TSS’). Transcov will help you with the following:

- Extract data from bam files
- Build a data structure
- Manipulating the data structure, like aggregating across samples
- Explore coverage around TSS’.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by Hogfeldt** [comment]
@BiocondaBot please add label

File: recipes/transcov/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - attrs
+    - click >=6.0
+    - numpy
+    - pip
+    - pysam
+    - python >=3.6
```
Comment (simonbray) [inline recipes/transcov/meta.yaml:26]: Do you need all these requirements in host? python and pip should be enough I'd have thought.

Comment (Hogfeldt) [reply recipes/transcov/meta.yaml:22]: No, I think they can be removed.

**Comment by simonbray** [comment]
Thanks for the contribution @Hogfeldt!

**Comment by simonbray** [comment]
@bioconda-bot please merge

**Pull request merged**